papermill/execute.py

```diff
@@ -240,6 +240,20 @@ def _form_escaped_value(str_val):
 # Registry for functions that build parameter assignment code.
 _parameter_code_builders = {}
 
+def _form_escaped_python_value(val):
```

Comment:
Let's consider renaming to `_translate_type_python`, `_translate_str_python`, `_translate_list_python`.... See https://github.com/nteract/papermill/pull/93#discussion_r161518590 for rationale.